corehq/apps/app_manager/models.py

```diff
@@ -986,9 +986,91 @@ def validate(self, value, required=True):
 
 
 class NavMenuItemMediaMixin(DocumentSchema):
+    """
```

Comment:
I think its still fine to use `default` in the migration and in `wrap` but after that I don't think its necessary to keep it updated if you use the same pattern as we do for form name translations.